### PR TITLE
Apply security policies

### LINKED ISSUE #137: Apply security policies
- [x] Content Security Policy (CSP) header
- [x] X-Content-Type-Options header
- [x] X-Frame-Options (XFO) header
- [x] X-XSS-Protection header

### PR DESCRIPTION
**Pull request by martinratinaud**
In order to have greater grades on dashlord scan, implement security policies.

With these, we manage to get the following grades

Screen Shot 2022-06-30 at 15 38 26

Which seems to be good enough.

I'm checking with scalingo why their https is giving a `B` grade only.
ANd the cookies seem to be put by matomo so there's nothing we can do about it

You can see the full test on our hosted version of dashlord https://opentermsarchive.github.io/dashlord/

Fixes #137

### CODE REVIEW

**Comment by martinratinaud** [comment]
@clementbiron I deployed the version already on opentermsarchive.org in order for it to be crawled by SocialGouv's dashlord (which will happen on sunday night)

I suggest you go through every page and re-test all functionalities to make sure I did not forget anything.
What you might be looking at would be
- images not showing
- weird javascript errors in the console
- css not applied

Thanks

**Review by Ndpnt — APPROVED** [review]

File: next.config.js [hunk 1]
```diff
@@ -66,4 +66,65 @@ module.exports = {
...
+        headers: [
+          {
+            key: 'Content-Security-Policy',
+            value: Object.entries(securityPolicies)
+              .map(([securityKey, values]) => `${securityKey} ${values.join(' ')}`)
+              .join('; '),
+          },
+          {
+            key: 'X-Frame-Options',
+            value: 'DENY',
```
Comment (MattiSG) [inline next.config.js:112]: I find it a bit sad that we now by default block embedding the contribution tool in an iframe 🙂

Comment (martinratinaud) [reply next.config.js:112]: Well unfortunately, the other possible value is `SAMEORIGIN` so it won't help.

We can remove it later on I guess, but for now does anybody tried to put ota.org in an iframe?

Comment (martinratinaud) [reply next.config.js:112]: Ok so let's keep it that way to maximize our score on dashlord

**Pull request merged**